### PR TITLE
HFP-3556 Missing accessible name for <nav> elements

### PR DESCRIPTION
**Pull request by zouhairloucif**

### CODE REVIEW

File: src/scripts/navigation-line.js [hunk 1]
```diff
@@ -449,8 +449,8 @@ const NavigationLine = (function ($) {
       .addClass("h5p-progressbar-part-selected")
       .attr('aria-selected', true)
       .siblings()
-        .removeClass("h5p-progressbar-part-selected")
-        .attr('aria-selected', false);
+      .removeClass("h5p-progressbar-part-selected")
+      .attr('aria-selected', false);
```
Comment (VildeStabell) [inline src/scripts/navigation-line.js:453]: Is there a reason you removed the indents here?

File: src/scripts/cp.js [hunk 2]
```diff
@@ -229,7 +230,7 @@ CoursePresentation.prototype.attach = function ($container) {
           '     <div class="h5p-slides-wrapper"></div>' +
           '    </div>' +
           '  </div>' +
-          '  <nav class="h5p-cp-navigation">' +
+          '  <nav class="h5p-cp-navigation" aria-labelledby="' + this.l10n.slideshowNavigationLabel + '">' +
```
Comment (VildeStabell) [inline src/scripts/cp.js:233]: `aria-labelledby` expects the id of the element containing the corresponding label, not the actual label. Consider using `aria-label` instead

**Review by VildeStabell — CHANGES_REQUESTED** [review]

File: src/scripts/navigation-line.js (hunk 1 above)
Comment (zouhairloucif) [reply src/scripts/navigation-line.js:453]: It's for the code formatting norme, you might be right since the indented code concerns only the siblings

Comment (VildeStabell) [reply src/scripts/navigation-line.js:453]: From [this example](https://contribute.jquery.org/style-guide/js/#chained-method-calls) it looks like there should be an extra level of indentation when the context changes.

**Review by VildeStabell — APPROVED** [review]

**Pull request merged**